Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1062,7 +1066,7 @@ - (void)updateSessionRestoreBanner {
     BOOL shouldDetachBanner = !isContactThread;
     if (isContactThread) {
         TSContactThread *thread = (TSContactThread *)self.thread;
-        if (thread.sessionRestoreDevices.count > 0) {
+        if (thread.sessionRestoreDevices.count > 0 && thread.sessionResetStatus == LKSessionResetStatusNone) {
```

Comment:
A change we just made recently on desktop is that we no longer ask the user if sessions need to be reset. Basically on restore if you can't decrypt or you get `no record for session` error then you send out a `SessionRequest` if you haven't done it before.